src/main/drivers/accgyro_mpu.c

```diff
@@ -215,15 +240,7 @@ extiCallbackRec_t mpuIntCallbackRec;
 void mpuIntExtiHandler(extiCallbackRec_t *cb)
```

Comment:
All this exti init stuff is done through entire code for all sensors. Too much mess to clean for now if I go that way. Checkout my latest commit what works